ee/clickhouse/queries/experiments/trend_experiment_result.py

```diff
@@ -34,12 +36,15 @@
 P_VALUE_SIGNIFICANCE_LEVEL = 0.05
 
 
-@dataclass(frozen=True)
+@dataclass
```

Comment:
why unfreeze this data class?  AFAICT we're not changing the value of these fields when calculating the credible intervals, so what's the benefit to unfreezing this?

Reply:
was testing something and forgot to freeze back 🙈 well spotted!